File: launchdarkly/typescript-axios/src/GetAuditLogEntryExample.ts
import axios, { AxiosError } from "axios";
import * as api from "launchdarkly_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

new api.AuditLogApi(configuration).getAuditLogEntry(
  "id_string", // id
).then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling AuditLogApi#getAuditLogEntry:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});

File: launchdarkly/ruby/src/GetAuditLogEntryExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

begin
    response = LaunchDarklyClient::AuditLogApi.new.get_audit_log_entry(
        nil, # id
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling AuditLogApi#get_audit_log_entry: #{e}"
end
